src/main/java/com/greencity/ui/component/ToolbarComponent.java

```diff
@@ -0,0 +1,40 @@
+package com.greencity.ui.component;
+
+import lombok.Getter;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class ToolbarComponent {
+    private WebDriver driver;
```

Comment:
removed this line private WebDriver driver; because you are inheriting it from `BaseComponent`